CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project `7.0.x` per each release will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v7.0.0.3] - 2020-07-03
+
+### Added
```

Comment:
@RidmiR better to add a `Fixed` section as in [this](https://keepachangelog.com/en/1.0.0/) recommendation and mention about the checksum issue.